pkg/runtime/errors/init.go

```diff
@@ -43,8 +43,8 @@ func (e *InitError) Unwrap() error {
 	return e.err
 }
 
-// NewInitError returns an InitError wrapping an existing context error.
-func NewInitError(kind InitErrorKind, entity string, err error) *InitError {
+// NewInit returns an InitError wrapping an existing context error.
+func NewInit(kind InitErrorKind, entity string, err error) *InitError {
```

Comment:
I would like the old one since it is more clear, why change it?

Reply:
This causes a stutter in the package function call:

```go
errors.NewInit
```

vs


```go
errors.NewInitError
```